src/Sulu/Bundle/WebsiteBundle/Controller/WebsiteController.php

```diff
@@ -161,4 +162,17 @@ protected function getCacheTimeLifeEnhancer(): ?CacheLifetimeEnhancer
 
         return $this->get('sulu_http_cache.cache_lifetime.enhancer');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedServices()
```

Comment:
this method should configure the minimal container that is passed to the controller. unfortunately apparently the method does not affect anything right now and the controller still receives the complete container 😢

Reply:
the only way that i found to solve this is adding `autowire: true` to the service definition of the container..

Reply:
see https://github.com/symfony/symfony/issues/33755

Reply:
@nnatter I think we should still merge this. But we should write into the `UPGRADE.md` that the services which are accessed need to define by adding an example code to the upgrade file.